castai/add-castai-removal-disabled/add-castai-removal-disabled.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: add-castai-removal-disabled
+  annotations:
+    policies.kyverno.io/title: Add CAST AI Removal Disabled
+    policies.kyverno.io/category: CAST AI
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Job, CronJob
+    kyverno.io/kyverno-version: 1.9
```

Comment:
This may need to be wrapped in double quotes, or alternatively use a patch version as well.

Reply:
Yup, needed to be quoted, submitting the change.